judge/comments.py

```diff
@@ -31,12 +31,15 @@ class Meta:
             'parent': forms.HiddenInput(),
         }
 
-        widgets['body'] = MartorWidget(attrs={'data-markdownfy-url': reverse_lazy('comment_preview')})
+        widgets['body'] = MartorWidget(
+            editor_msg=_('Please click on "Preview" before posting your comment.'),
+            button_text=_('Post!'),
+            attrs={'data-markdownfy-url': reverse_lazy('comment_preview')},
+        )
 
     def __init__(self, request, *args, **kwargs):
         self.request = request
         super(CommentForm, self).__init__(*args, **kwargs)
-        self.fields['body'].widget.attrs.update({'placeholder': _('Comment body')})
```

Comment:
Just to confirm, is this dead code? I see it's being removed without anything similar being added.

Reply:
yes, this became dead when dmoj migrated from pagedown to martor